src/ASM/ASMs2D.C

```diff
@@ -1297,44 +1297,40 @@ bool ASMs2D::updateCoords (const Vector& displ)
 }
 
 
-void ASMs2D::getBoundaryNodes (int lIndex, IntVec& nodes) const
+void ASMs2D::getBoundaryNodes (int lIndex, IntVec& nodes, int basis) const
 {
-  if (!surf) return; // silently ignore empty patches
+  if (basis == 0)
+    basis = 1;
+
+  if (!getBasis(basis)) return; // silently ignore empty patches
```

Comment:
rettelse: du får ut alle basiser fra mixed, men hvis ASMs2D blir kalt direkte med 0 resettes den til 1 siden det er den eneste basisen vi har.

ref ASMs2Dmx::getBoundaryNodes.

så doxy på ASMs2D er feil, den gjelder kun for mixed-overstyrelsen.